ps_viewedproduct.php

```diff
@@ -168,8 +168,8 @@ public function getCacheId($name = null)
 
     public function renderWidget($hookName = null, array $configuration = [])
     {
-        if ('displayProductButtons' === $hookName) {
```

Comment:
(I was writing another review, then I found the actual issue)

On PrestaShop 1.7.1.0, we declared an alias for the hook 'displayProductButtons'.
The only you have to replace in this module is this line with the following one:

```php
if (in_array($hookName, array('displayProductButtons', 'displayProductAdditionalInfo')))
```